### PR TITLE
Global error enhancements

### PR DESCRIPTION
**Pull request by msecret**
- Modified the message so it optionally show a cloud.gov status
link if one exists, so user can check if cloud.gov is down.
- Changed the messaging a bit based on usability testing.
- Remove the loading indicator in case of errored app stats
fetch request.

### CODE REVIEW

File: static_src/actions/error_actions.js [hunk 1]
```diff
@@ -38,10 +38,10 @@ export default {
   },
 
   importantDataFetchError(err, entityMessage) {
-    let msg = 'Page failed to load, please try again';
+    let msg = 'Connection issue, please try again';
```
Comment (adborden) [inline static_src/actions/error_actions.js]: This is a little terse, maybe something more like:
> There was an issue connecting to dashboard.cloud.gov, please try again later.

File: static_src/components/global_error.jsx [hunk 2]
```diff
@@ -35,10 +36,21 @@ export default class GlobalError extends React.Component {
 
   render() {
     const err = this.props.err;
+    const link = (config.docs.status) && (
+      <span> check <a target="_blank" href={ config.docs.status }>
+        cloud.gov's status</a> or
```
Comment (adborden) [inline static_src/components/global_error.jsx]: We want to avoid hard-coding "cloud.gov" here, maybe:
> ... check [the platform's status] or ...

Comment (msecret) [inline static_src/components/global_error.jsx]: oh yea forgot, good catch.

File: static_src/actions/error_actions.js (hunk 1 above)
Comment (msecret) [inline static_src/actions/error_actions.js]: I like that, but from the user testing, it seems like the specific messaging was useful, so I want to  keep the "app usage data might not be available" in there.

**Pull request merged**